draft-edm-protocol-greasing.md

```diff
@@ -355,9 +356,13 @@ connections, or phases within a connection.
 
 # Considerations for Protocol Versions {#versions}
 
-There are intrinsic and well-documented issues related to testing version
+{{TRANSITION}} discusses considerations around planning for transitioning
```

Comment:
Unsure if planned or coincidence but this text is in a section mooted for removal in https://github.com/intarchboard/draft-protocol-greasing/issues/49#issue-3249802793

Reply:
I don't think it should be removed.